src/lava/magma/core/model/py/model.py

```diff
@@ -130,7 +141,7 @@ def _set_var(self):
 
         # 2. Receive Var data
         data_port = self.service_to_process
-        if isinstance(var, int) or isinstance(var, np.integer):
+        if isinstance(var, int) or isinstance(var, np.int32):
```

Comment:
Please replace `np.int32` with `np.int` - the aliases have been deprecated and will produce deprecation warnings.
https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

Reply:
I think it's the other way round. `np.int` is deprecated. At least I get a warning on usage.